### DISCUSSION TITLE
[WS-1271] Nav tab highlights wrong tab

### DISCUSSION THREAD

**Discussion by real-monster** [vercel/community#6042] [Bug Report]
### Page to Investigate

https://vercel.com/dashboard

### Steps to Reproduce

Browser: Chrome
Platform: macOS

1. Open https://vercel.com/dashboard
2. Hold the "command" key down while clicking on any of the other tabs in the nav to open the page in a new tab.
3. The nav highlights the clicked tab even though the current tab has not changed pages.

View screenshot below.
<img width="1430" alt="Screenshot of bug" src="https://github.com/vercel/community/assets/110545391/1ba77a18-d207-469c-99fc-e2258a6b3acd">

**Comment by swarnava** [comment]
Hi,

Thanks for reporting this. I have reported this to relevant team! :)
